include/nbl/asset/utils/IShaderCompiler.h

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_ASSET_I_SHADER_COMPILER_H_INCLUDED__
+#define __NBL_ASSET_I_SHADER_COMPILER_H_INCLUDED__
```

Comment:
things to fix going forward, remove `__` from header guards, its reserved for system/compiler preprocessor defines apparently

Reply:
So it will be `NBL_ASSET_I_SHADER_COMPILER_H_INCLUDED`?

Reply:
no, `_NBL_ASSET_I_SHADER_COMPILER_H_INCLUDED_`